syntax = "proto3";

message Statement{
  string type = 1;
}

message Entity {
  string type = 1;
  repeated string values = 2;
}

message EntityResponse {
  repeated Entity entities = 1;
}

service MainService{
  // Sends a greeting
  rpc GetKeywords(Statement) returns (EntityResponse);
}
